rkt/run.go

```diff
@@ -227,6 +227,8 @@ func runRun(cmd *cobra.Command, args []string) (exit int) {
 		pcfg.Ports = []types.ExposedPort(flagPorts)
 		pcfg.InheritEnv = flagInheritEnv
 		pcfg.ExplicitEnv = flagExplicitEnv.Strings()
+		// keep these env variables since systemd uses them for socket activation
+		pcfg.KeepEnv = []string{"LISTEN_FDS", "LISTEN_PID"}
```

Comment:
It can be problem here, but I'm not sure - IIRC `LISTEND_FDS` and `LISTEN_PID` are way to pass file descriptors from "internal systemd to application" rather than "external' (host) systemd.

What should work for "common" env variables, I think will not work for "socket activation" variables.

Your variables has to go through all layers like that.

```
# running
systemd(host) -> rkt(binary) -> init -> systemd(internal) -> appexec -> app
# entering
rkt(binary) -> enter -> ssh(kvm)/nsenter -> appexec -> app
```

what I'm worried about is the `LISTEN_*` values are copied to early. Deeper analysis is needed.

Maybe we should, had list names of variables (without values) and each layer is responsible for passing it down (sometimes thorough file and sometimes just by inheriting and sometimes by `send_env` in ssh options) and then if some layer like "internal systemd" can override what it wants (but it sounds a bit like code duplication).